build

```diff
@@ -20,9 +20,12 @@ for d in networking/plugins/*; do
 	go install ${REPO_PATH}/$d
 done
 
+echo "Building rkt (stage0)..."
+go build -o $GOBIN/rkt ${REPO_PATH}/rkt
+
 if [[ "$OSTYPE" == "linux-gnu" ]]; then
 	echo "Building init (stage1)..."
-	go build -o $GOBIN/init ${REPO_PATH}/stage1/init
+	go install ${REPO_PATH}/stage1/init
```

Comment:
why change this to install but not the actual rkt build? (I don't have a strong preference either way, but curious as to whether the difference is intentional)